### ISSUE TITLE
Incompatible with Requests 0.14.0

### ISSUE THREAD

**Issue by bradmontgomery** [copitux/python-github3#26]
It looks like something in Requests 0.14.0 (maybe ealier, I've been using 0.13.2) has broken the way python-github3 handles session params. 

Creating a `Github` object fails with the following traceback:

``` python
Python 2.7.1 (r271:86832, Jun 25 2011, 05:09:01) 
[GCC 4.2.1 (Based on Apple Inc. build 5658) (LLVM build 2335.15.00)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import requests
>>> requests.__version__
'0.14.0'
>>> from pygithub3 import Github
>>> gh = Github()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "pygithub3/github.py", line 24, in __init__
    self._users = User(**config)
  File "pygithub3/services/users/__init__.py", line 14, in __init__
    self.keys = Keys(**config)
  File "pygithub3/services/base.py", line 44, in __init__
    self._client = Client(**config)
  File "pygithub3/core/client.py", line 29, in __init__
    self.__set_params(self.config)
  File "pygithub3/core/client.py", line 56, in __set_params
    self.requester.params['per_page'] = config.get('per_page')
TypeError: list indices must be integers, not str
```

**Comment by copitux** [comment]
Yes, a contract was broken with requests >0.14.0. @ralphbean [detected it](https://github.com/ralphbean/bugwarrior/issues/31#issuecomment-8070249) a few weeks ago.

This summer has been very hard to me, I'm going to continue the development this weekend and next week.

Update that lib to support new requests module, study and merge some pull requests, cover new API requests... and so on

Sorry for the delay :S

**Comment by bradmontgomery** [comment]
Completely Understand :)  I'm travelling this week so have limited time to dig into extra things. I appreciate your work maintaining this!

**Comment by copitux** [comment]
solved: 6fe57cf

**Comment by ralphbean** [comment]
+1
